visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.tsx

```diff
@@ -466,15 +566,15 @@ export class CohortEditor extends React.PureComponent<ICohortEditorProps, ICohor
                                     min={selectedMeta.featureRange.min}
                                     max={selectedMeta.featureRange.max}
                                     value={openedFilter.arg.toString()}
-                                    onIncrement={this.setNumericValue.bind(this, numericDelta, selectedMeta)}
-                                    onDecrement={this.setNumericValue.bind(this, -numericDelta, selectedMeta)}
-                                    onValidate={this.setNumericValue.bind(this, 0, selectedMeta)}
+                                    onIncrement={this.setNumericValue.bind(this, numericDelta, selectedMeta, 0)}
```

Comment:
it looked like this was defaulted to 0? If it is just made required and set here, then defintely don't need the check on method === in the range